tests/e2e/edgesite/edgesite_suite_test.go

```diff
@@ -45,9 +45,8 @@ func TestEdgecoreAppDeployment(t *testing.T) {
 		nodeName = "integration-node-" + utils.GetRandomString(10)
 		nodeSelector = "node-" + utils.GetRandomString(3)
 		Expect(utils.DeploySetup(ctx, nodeName, "edgesite")).Should(BeNil())
-		Expect(utils.StartEdgeSite()).Should(BeNil())
-
 		Expect(utils.RegisterNodeToMaster(nodeName, ctx.Cfg.K8SMasterForKubeEdge+constants.NodeHandler, nodeSelector)).Should(BeNil())
```

Comment:
Auto register edge node in default, we can remove this line?

Reply:
It creates node with a node-selector label so that all the pods for the testcase are scheduled on this node. Since we have common setup, it maybe required, once we move to dynamic setup we can remove it.